Index: src/Application.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Application.php b/Src/Application.php
rename from src/Application.php
rename to Src/Application.php
--- a/src/Application.php	(revision 3e5aafb29af590c76d26ffd41af85878065418c7)
+++ b/Src/Application.php	(date 1677450856551)
@@ -8,8 +8,8 @@
 use \App\Core\Routing\Router;
 
 require_once __DIR__ . '/../config/Config.php';
-//require_once __DIR__ . '/../core/Database/Migration/Migrator.php';
-//require_once __DIR__ . '/../core/Database/Service/DBSession.php';
+//require_once __DIR__ . '/../Core/Database/Migration/Migrator.php';
+//require_once __DIR__ . '/../Core/Database/Service/DBSession.php';
 
 
 class Application
@@ -73,6 +73,7 @@
 			}
 
 			$relativePath = str_replace($prefix, "/", $class);
+            $relativePath = str_replace('\\', "/", $relativePath);
 
 			$file = $base_dir . $relativePath . '.php';
 			if (!file_exists($file))
Index: core/Database/Migration/Migrator.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/core/Database/Migration/Migrator.php b/Core/Database/Migration/Migrator.php
rename from core/Database/Migration/Migrator.php
rename to Core/Database/Migration/Migrator.php
--- a/core/Database/Migration/Migrator.php	(revision 3e5aafb29af590c76d26ffd41af85878065418c7)
+++ b/Core/Database/Migration/Migrator.php	(date 1677450918771)
@@ -8,7 +8,7 @@
 
 class Migrator
 {
-	protected static string $pathToMigrationFolder = Config::ROOT . "/src/Migration/";
+	protected static string $pathToMigrationFolder = Config::ROOT . "/Src/Migration/";
 
 	public static function migrate(): void
 	{
@@ -29,7 +29,7 @@
 			$lastMigration = null;
 		}
 
-		// 2. поиск всех новых миграций в /src/Migration/
+		// 2. поиск всех новых миграций в /Src/Migration/
 		$allMigrations = array_diff(scandir(self::$pathToMigrationFolder), ['..', '.']);
 
 		if (!$allMigrations)
Index: public/index.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\n\nrequire_once '../src/Application.php';\n\n\\App\\Application::run();
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/public/index.php b/public/index.php
--- a/public/index.php	(revision 3e5aafb29af590c76d26ffd41af85878065418c7)
+++ b/public/index.php	(date 1677450918751)
@@ -1,5 +1,5 @@
 <?php
 
-require_once '../src/Application.php';
+require_once '../Src/Application.php';
 
 \App\Application::run();
\ No newline at end of file
Index: src/View/Main/index.html.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/View/Main/index.html.php b/Src/View/Main/index.html.php
rename from src/View/Main/index.html.php
rename to Src/View/Main/index.html.php
--- a/src/View/Main/index.html.php	(revision 3e5aafb29af590c76d26ffd41af85878065418c7)
+++ b/Src/View/Main/index.html.php	(date 1677450918767)
@@ -20,7 +20,7 @@
 		</a>
 
         <div class="div-search">
-            <img src="/src/View/icons/icon-search.svg" class="icon-search"></img>
+            <img src="/Src/View/icons/icon-search.svg" class="icon-search"></img>
             <label>
                 <input type="text" class="search" placeholder="Поиск по товарам">
             </label>
Index: src/Controller/BaseController.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Controller/BaseController.php b/Src/Controller/BaseController.php
rename from src/Controller/BaseController.php
rename to Src/Controller/BaseController.php
--- a/src/Controller/BaseController.php	(revision 3e5aafb29af590c76d26ffd41af85878065418c7)
+++ b/Src/Controller/BaseController.php	(date 1677450918759)
@@ -13,7 +13,7 @@
 			throw new PathException('invalid page address');
 		}
 
-		$path = Config::ROOT . "/src/View/$path.php";
+		$path = Config::ROOT . "/Src/View/$path.php";
 
 		extract($args, EXTR_OVERWRITE);
 
diff --git a/core/Routing/Router.php b/Core/Routing/Router.php
rename from core/Routing/Router.php
rename to Core/Routing/Router.php
diff --git a/core/Routing/Route.php b/Core/Routing/Route.php
rename from core/Routing/Route.php
rename to Core/Routing/Route.php
diff --git a/src/DAO/ImageDAO.php b/Src/DAO/ImageDAO.php
rename from src/DAO/ImageDAO.php
rename to Src/DAO/ImageDAO.php
diff --git a/src/DAO/GoodDAO.php b/Src/DAO/GoodDAO.php
rename from src/DAO/GoodDAO.php
rename to Src/DAO/GoodDAO.php
diff --git a/src/DAO/OrderDAO.php b/Src/DAO/OrderDAO.php
rename from src/DAO/OrderDAO.php
rename to Src/DAO/OrderDAO.php
diff --git a/src/DAO/TagDAO.php b/Src/DAO/TagDAO.php
rename from src/DAO/TagDAO.php
rename to Src/DAO/TagDAO.php
diff --git a/src/View/Auth/auth.html.php b/Src/View/Auth/auth.html.php
rename from src/View/Auth/auth.html.php
rename to Src/View/Auth/auth.html.php
diff --git a/src/View/Good/good.html.php b/Src/View/Good/good.html.php
rename from src/View/Good/good.html.php
rename to Src/View/Good/good.html.php
diff --git a/src/View/Order/orderRegistration.html.php b/Src/View/Order/orderRegistration.html.php
rename from src/View/Order/orderRegistration.html.php
rename to Src/View/Order/orderRegistration.html.php
diff --git a/src/View/Order/orderPlaced.html.php b/Src/View/Order/orderPlaced.html.php
rename from src/View/Order/orderPlaced.html.php
rename to Src/View/Order/orderPlaced.html.php
diff --git a/src/View/Detail/detail.html.php b/Src/View/Detail/detail.html.php
rename from src/View/Detail/detail.html.php
rename to Src/View/Detail/detail.html.php
diff --git a/src/View/service/404.html.php b/Src/View/service/404.html.php
rename from src/View/service/404.html.php
rename to Src/View/service/404.html.php
diff --git a/src/Model/Order.php b/Src/Model/Order.php
rename from src/Model/Order.php
rename to Src/Model/Order.php
diff --git a/src/View/service/fatal.html.php b/Src/View/service/fatal.html.php
rename from src/View/service/fatal.html.php
rename to Src/View/service/fatal.html.php
diff --git a/src/Model/Good.php b/Src/Model/Good.php
rename from src/Model/Good.php
rename to Src/Model/Good.php
diff --git a/src/Model/Customer.php b/Src/Model/Customer.php
rename from src/Model/Customer.php
rename to Src/Model/Customer.php
diff --git a/src/Model/Tag.php b/Src/Model/Tag.php
rename from src/Model/Tag.php
rename to Src/Model/Tag.php
diff --git a/src/Model/Image.php b/Src/Model/Image.php
rename from src/Model/Image.php
rename to Src/Model/Image.php
diff --git a/src/Model/User.php b/Src/Model/User.php
rename from src/Model/User.php
rename to Src/Model/User.php
diff --git a/src/Service/GoodService.php b/Src/Service/GoodService.php
rename from src/Service/GoodService.php
rename to Src/Service/GoodService.php
diff --git a/src/Service/IndexService.php b/Src/Service/IndexService.php
rename from src/Service/IndexService.php
rename to Src/Service/IndexService.php
diff --git a/core/Database/Service/DBSession.php b/Core/Database/Service/DBSession.php
rename from core/Database/Service/DBSession.php
rename to Core/Database/Service/DBSession.php
diff --git a/src/Service/InvalidInputException.php b/Src/Service/InvalidInputException.php
rename from src/Service/InvalidInputException.php
rename to Src/Service/InvalidInputException.php
diff --git a/src/Service/OrderService.php b/Src/Service/OrderService.php
rename from src/Service/OrderService.php
rename to Src/Service/OrderService.php
diff --git a/src/Migration/2023_01_31_19_30_create_migration_table.sql b/Src/Migration/2023_01_31_19_30_create_migration_table.sql
rename from src/Migration/2023_01_31_19_30_create_migration_table.sql
rename to Src/Migration/2023_01_31_19_30_create_migration_table.sql
diff --git a/src/Migration/2023_02_09_22_25_update_changing_column_names_specifically_for_Pavel_Volkov.sql b/Src/Migration/2023_02_09_22_25_update_changing_column_names_specifically_for_Pavel_Volkov.sql
rename from src/Migration/2023_02_09_22_25_update_changing_column_names_specifically_for_Pavel_Volkov.sql
rename to Src/Migration/2023_02_09_22_25_update_changing_column_names_specifically_for_Pavel_Volkov.sql
diff --git a/src/Migration/2023_02_08_18_30_update_order_and_good_table.sql b/Src/Migration/2023_02_08_18_30_update_order_and_good_table.sql
rename from src/Migration/2023_02_08_18_30_update_order_and_good_table.sql
rename to Src/Migration/2023_02_08_18_30_update_order_and_good_table.sql
diff --git a/src/Migration/2023_02_01_17_00_create_all_main_tables.sql b/Src/Migration/2023_02_01_17_00_create_all_main_tables.sql
rename from src/Migration/2023_02_01_17_00_create_all_main_tables.sql
rename to Src/Migration/2023_02_01_17_00_create_all_main_tables.sql
diff --git a/src/Controller/IndexController.php b/Src/Controller/IndexController.php
rename from src/Controller/IndexController.php
rename to Src/Controller/IndexController.php
diff --git a/src/Controller/GoodController.php b/Src/Controller/GoodController.php
rename from src/Controller/GoodController.php
rename to Src/Controller/GoodController.php
diff --git a/src/Controller/OrderController.php b/Src/Controller/OrderController.php
rename from src/Controller/OrderController.php
rename to Src/Controller/OrderController.php
